pkg/local_object_storage/blobovnicza/get.go

```diff
@@ -1,6 +1,7 @@
 package blobovnicza
 
 import (
+	"github.com/nspcc-dev/neo-go/pkg/util/slice"
```

Comment:
is it ok to use such util from `neo-go` in the engine of the node?

maybe implement somewhere in that repo?

Reply:
Why not? It is not that we add a new dependency like in the `api-go` case.